pkgs/misc/drivers/imagescan/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchgit, pkgconfig, patchelf, boost, udev, libusb, imagemagick
+, sane-backends, gtkmm2 }:
+
+stdenv.mkDerivation rec {
+  name = "imagescan-${version}";
+  version = "3.59.2";
+  src = fetchgit {
```

Comment:
Please use ``fetchFromGitHub```.

Reply:
Even though the project is __not__ on github but on gitlab?
https://github.com/utsushi/imagescan has been archived and points towards gitlab.

Reply:
There is also a `fetchFromGitLab` function.

Reply:
Thanks, I didn't knew that! I adjusted accordingly.